tests/queries/0_stateless/03734_correlated_subqueries_reorder.reference

```diff
@@ -0,0 +1,118 @@
+Expression ((Project names + Projection))
```

Comment:
I don't see this plan has ReadFromCommonBufferSource or something about in-memory buffers ...